pkgs/tools/networking/glorytun/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, autoreconfHook
+, libsodium
+, iproute2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "glorytun";
+  version = "0.3.4";
+
+  src = fetchFromGitHub {
+    owner = "angt";
+    repo = pname;
+    rev = "v${version}";
```

Comment:
```suggestion
    repo = finalAttrs.pname;
    rev = "v${finalAttrs.version}";
```

Reply:
no, we should never use `finalAttrs.pname` in a url